src/devtools/client/debugger/src/utils/editor/get-token-location.js

```diff
@@ -3,16 +3,3 @@
  * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
 
 //
```

Comment:
Can we delete this file then?

Reply:
Oh yeah, I should have done that earlier! :)